packages/next/package.json

```diff
@@ -95,7 +95,7 @@
     "@next/env": "14.0.2-canary.3",
     "@swc/helpers": "0.5.2",
     "busboy": "1.6.0",
-    "caniuse-lite": "^1.0.30001406",
+    "caniuse-lite": "^1.0.30001559",
```

Comment:
Should this be pinned to the version instead of using the `^` range operator?

Reply:
The `^` lets the user can install newer versions. So I'm not sure htis PR is actually needed. Regardless, I created a new PR since this one has merge conflicts as is already outdated since there is a new version

https://github.com/vercel/next.js/pull/60827